maintainers/maintainer-list.nix

```diff
@@ -14009,6 +14009,12 @@
     githubId = 5978566;
     name = "Yves-Stan Le Cornec";
   };
+  ylh = {
```

Comment:
Two things:

1. If you do not plan to be one of the `meta.maintainers` of this package, this is mostly unneeded.
2. Regardless of this, you should put this in a separated commit with the title `maintainers: add myself`

Reply:
For 1. see line 94

Reply:
Thanks. For some reason I have not seen it before.
Nonetheless 2 remains.

Please @ylh split the commit into two.